_posts/2025-01-04-glorias-brink-epoch.md

```diff
@@ -0,0 +1,154 @@
+paths including a coveted job offer to join Google full time after interning
+there the summer prior. Instead of taking the safe career path, she took a
+chance on Bitcoin development, and thanks to the visionary support of our
+[founding sponsors](/sponsors)—and early contributions from the
+[Human Rights Foundation and
+Spiral](/blog/2020/12/01/gloria/)—joined Brink as our first grantee.
+
+## A Package Relay Journey
+
+During Gloria's first year as a Brink fellow, she was mentored by Brink
+co-founder John Newbery and moved to London to establish our office there. After
+immersing herself in the Bitcoin Core codebase and coming up with her
+[fellowship project
+](/blog/2021/01/21/fellowship-project-package-accept/)in early
+2021, she was ready to put forth the project to the community with her [Package
+Mempool Accept and Package RBF
+proposal](https://gnusha.org/pi/bitcoindev/CAFXO6=+cHyQKM2n9yn4EhwLZO+AUB0ZD81qWPxmpN27rjUoU3w@mail.gmail.com/)
+in September of 2021.
+
+Commenting on the motivation for her work these past 4 years, Gloria notes:
+
+“*I work in a problem-oriented fashion, so all of the other mempool policy
+proposals stem from the core problem of fixing pinning attacks in exogenous fee
+bumping, which is a crucial tool in contracting protocols. Package Relay is
+maybe the “flagship” proposal and certainly the biggest chunk of work, but the
+name is often a catch-all for anti-pinning proposals including TRUC.*”
+
+Recognizing her momentum and expertise, Brink brought Gloria on as a grantee in
+2022 to continue her work. That year she continued her P2P research, expanded
+the work to include fee bumping more generally, posted a discussion on RBF
+improvements to the [mailing
+list](https://gnusha.org/pi/bitcoindev/CAFXO6=LGbaur6XQrE+6a6mAAHXduOCXoWPTgPosxAG59ZkK6Gg@mail.gmail.com/),
+and posted a P2P protocol proposal which later became [BIP 331](https://github.com/bitcoin/bips/blob/master/bip-0331.mediawiki) to the [mailing
+list](https://gnusha.org/pi/bitcoindev/CAFXO6=JROe_9ih2h+_CCH-UbxehsM5RQ6YyNnPesEpveBEtdow@mail.gmail.com/)
+May 2022. That same year, she was named a Bitcoin Core maintainer by her peers,
+an honor that speaks volumes about her technical skill and collaborative spirit.
+
+In 2023, Brink continued support for Gloria and her work and, recognizing the
+valuable technical perspectives she has on the Bitcoin Core project, also
+invited her to join our grant committee to help evaluate funding applications.
+Meanwhile, Gloria continued her work on package relay and v3 transaction
+relay/TRUC, projects that would eventually encompass a whopping 53 pull requests
+over the years. She also worked to achieve buy-in on her projects from her
+peers, resulting in the package relay project making the top priority projects
+list for the Bitcoin Core project for three consecutive releases (26.0, 27.0,
+28.0).
+
+Much of Gloria’s work was included in the recent [Bitcoin Core 28.0
```

Comment:
Maybe start this paragraph with something about 2024 being the year we saw the fruits of the labor, to make it clearer we have 1 paragraph per year

Reply:
Good idea, updated